com.pinegrow.Pinegrow.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.pinegrow.Pinegrow
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: pinegrow
+separate-locales: false
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+modules:
+
+  - name: pinegrow
+    buildsystem: simple
+    build-commands:
+      - install -D apply_extra /app/bin/apply_extra
+      - install -D pinegrow /app/bin/pinegrow
+      - install -Dm644 -t /app/share/metainfo com.pinegrow.Pinegrow.appdata.xml
+      - install -Dm644 -t /app/share/applications com.pinegrow.Pinegrow.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps com.pinegrow.Pinegrow.svg
+    sources:
+      - type: extra-data
+        only-arches:
+          - x86_64
```

Comment:
this doesn't address the problem since you don't have a source for aarch64.

See https://github.com/flathub/flathub/wiki/App-Maintenance#flathubjson

Reply:
Added the flathub.json, let me know if there is something else. Also fixed the icon issue